File: 프로그래머스/1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.js
function solution(lottos, win_nums) {
    let answer = 0;
    let cnt = 0
    for (let i=0; i<lottos.length; i++){
        if(win_nums.includes(lottos[i])){
            answer++;
        }else if(lottos[i] == 0){
            cnt++;
        }
    }
    let rank = {6:1,5:2,4:3,3:4,2:5,1:6,0:6};
    
    return [rank[answer+cnt],rank[answer]];
}

File: 프로그래머스/1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.py
def solution(lottos, win_nums):
    answer = 0
    cnt = 0
    rank = [6,6,5,4,3,2,1]
    for i in lottos:
        if i in win_nums:
            answer+=1
        elif i == 0:
            cnt += 1
    return [rank[answer+cnt],rank[answer]]